telegram/_bot.py

```diff
@@ -6602,7 +6603,7 @@ async def send_poll(
         is_anonymous: Optional[bool] = None,
         type: Optional[str] = None,  # pylint: disable=redefined-builtin
         allows_multiple_answers: Optional[bool] = None,
-        correct_option_id: Optional[int] = None,
+        correct_option_id: Optional[Literal[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]] = None,
```

Comment:
same suggestion as with the other PR: doing something like
```py
PossibleOptionID = Literal[0, 1 ....]
```
and same for Markdown version